screens/TabThreeScreen.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState, useEffect } from 'react';
+import { Button, Image, View, Platform } from 'react-native';
+import * as ImagePicker from 'expo-image-picker';
+
+export default function TabThreeScreen() {
+  const [image, setImage] = useState<string | null>(null);
+
+  useEffect(() => {
```

Comment:
Instead of requesting permission as soon as the user enters the screen, it would be better to request permission when the user clicks the button